### PR TITLE
NRPT-579: Fixed timezone issue in import BCOGC

### PR DESCRIPTION
**Pull request by danieltruong**

### CODE REVIEW

File: api/src/integrations/bcogc/inspections-utils.js [hunk 1]
```diff
@@ -41,7 +42,15 @@ class Inspections extends BaseRecordUtils {
     inspection['_sourceRefOgcDeficiencyId'] = csvRow['deficiency objectid'] || null;
 
     inspection['recordType'] = 'Inspection';
-    inspection['dateIssued'] = csvRow['inspection date'] || null;
+    let inspectionDate = null;
```
Comment (BcGovNeal) [inline api/src/integrations/bcogc/inspections-utils.js:45]: Could just do `inspection['dateIssued'] = csvRow['inspection date'] ? moment.tz(csvRow['inspection date'], "DD-MMM-YYYY", "America/Vancouver").toDate() : null;` here to make it simpler

**Review by BcGovNeal — APPROVED** [review]

File: api/src/integrations/bcogc/inspections-utils.js (hunk 1 above)
Comment (danieltruong) [reply api/src/integrations/bcogc/inspections-utils.js:45]: I had that originally! But it didn't seem to work at the time so I thought it had to do with the month being in all caps. I probably didn't restart my API :man_facepalming:.

Comment (danieltruong) [reply api/src/integrations/bcogc/inspections-utils.js]: 46c624a

Comment (BcGovNeal) [reply api/src/integrations/bcogc/inspections-utils.js]: Use nodemon!

**Review by mtCarto — APPROVED** [review]

**Pull request merged**